Testscripts/Windows/PERF-XDP-TCP-LATENCY.ps1

```diff
@@ -0,0 +1,209 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM and verifies there are no regression in Network Latency caused
+    by XDP. We achieve this by comparing lagscope results.
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+$iFaceName = "eth1"
+# Threshold value (40%) is calculated by analyzing 10 samples of latency values with and w/o XDP
+$thresholdValue = 1.4

```

Comment:
Do you see this threshold value change in future? If so, it is good to keep in replace XML file.

Reply:
If it is changed in the future (not that I know if it is going to be) we can change this parameter value in the script. As Simon mentioned above we should minimize XML parameters that are not frequently changed and could easily be changed in the script.